### ISSUE TITLE
Homework 1

### ISSUE THREAD

**Issue by lgatto** [lgatto/learning-git-github#7]
I will be posting small assignments as issues to help you with your git/github learning. To facilitate the homework announces, could you all *watch all activity* on this repository to get a notification when a new issue/homework is posted

![2022-10-04-152856_513x461_scrot](https://user-images.githubusercontent.com/384198/193832195-f5a89d20-59fb-4b4f-a8c8-0a067d6e6694.png)

Once this is done, give this post a thumbs up.
